Glimpse.ApplicationInsights/GlimpseTelemetryChannel.cs

```diff
@@ -76,16 +116,15 @@ public void Send(ITelemetry item)
                     FromLast = CalculateFromLast(timer),
                     IndentLevel = 0
                 };
-
                 MessageBroker.Publish(model);
             }
 
-            messageBroker.Publish(item);
-        }
+            if (!TelemetryConfiguration.Active.InstrumentationKey.ToString().Equals("00000000-0000-0000-0000-000000000000"))
```

Comment:
Do not check Active - check InstrumentationKey of an actual item that you are about to send